benefits/core/analytics.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, request, event_type, **kwargs):
 
         agency = session.agency(request)
         agency_name = agency.long_name if agency else None
-        verifier = session.verifier(request)
-        verifier_name = verifier.name if verifier else None
+        flow = session.flow(request)
+        verifier_name = flow.name if flow else None
```

Comment:
Would we want to use `flow_name` instead of `verifier_name`? Or maybe we decided to use this name since the analytics will still send `eligibility_verifier`/`verifier_name` key/values in `update_event_properties` and `update_user_properties`. It doesn't affect any functionality but I was wondering about this line.

Reply:
Yeah, I think it needs to stay `verifier_name` so that existing analytics charts still work and then https://github.com/cal-itp/benefits/issues/2248 will take care of updating it.

Reply:
Yup, purposefully ignored the analytics code in this PR, to wait for #2248